server/api/admin/users.js

```diff
@@ -38,3 +38,21 @@ router.delete('/:id', async (req, res, next) => {
     next(err)
   }
 })
+
+router.put('/:id/triggerresetpass', async (req, res, next) => {
+  try{
+    const user = await User.findById(req.params.id);
+    if (!user) {
+        console.log('No such user found:', req.body.email)
+        res.status(401).send('Wrong username and/or password')
```

Comment:
instead of logging this, you can simply call next(), which will go to the 404 not found route

Reply:
That's how they handle if user is not found in the boilermaker.